docs/reference/commandline/daemon.md

```diff
@@ -898,6 +900,8 @@ This is a full example of the allowed configuration options in the file:
 	"cluster-store": "",
 	"cluster-store-opts": [],
 	"cluster-advertise": "",
+	"max-concurrency-download": 3,
+	"max-concurrency-upload": 5,
```

Comment:
@MHBauer: I meant changing it at runtime by having the daemon reload the config file.